docs/user-guide/federation/federated-services.md

```diff
@@ -0,0 +1,234 @@
+
+The status of your Federated Service will automatically reflect the real-time status of the underlying Kubernetes services, for example:
+
+```
+kubectl --context=federation-cluster describe services nginx
+```
+
+```
+Name:                   nginx
+Namespace:              default
+Labels:                 run=nginx
+Selector:               run=nginx
+Type:                   LoadBalancer
+IP:         
+LoadBalancer Ingress:   104.197.246.190, 130.211.57.243, 104.196.14.231, 104.199.136.89, ...
+Port:                   http    80/TCP
+Endpoints:              <none>
+Session Affinity:       None
+No events.
+```
+
+Note the the 'LoadBalancer Ingress' addresses of your Federated Service correspond with the 'LoadBalancer Ingress' addresses of all of the underlying Kubernetes services (once these have been allocated - this may take a few seconds).  For inter-cluster and inter-cloud-provider networking between service shards to work correctly, your services need to have an externally visible IP address.  Service Type: Loadbalancer is typically used for this.
+
+Note also what we have not yet provisioned any backend Pods to receive the network traffic directed to these addresses (i.e. 'Service Endpoints'), so the Federated Service does not yet consider these to be healthy service shards, and has accordingly not yet added their addresses to the DNS records for this Federated Service (more on this aspect later). 
+
+## Adding Backend Pods
+
+To render the underlying service shards healthy, we need to add backend Pods behind them.  This is currently done directly against the API endpoints of the underlying clusters (although in future the Federation server will be able to do all this for you with a single command, to save you the trouble).  For example, to create backend Pods in 13 underlying clusters:
+
+```
+for CLUSTER in asia-east1-c asia-east1-a asia-east1-b \
+                        europe-west1-d europe-west1-c europe-west1-b \
+                        us-central1-f us-central1-a us-central1-b us-central1-c \
+						us-east1-d us-east1-c us-east1-b
+do
+  kubectl --context=$CLUSTER run nginx --image=nginx:1.11.1-alpine --port=80
+done
+```
+
+## Verifying Public DNS Records
+
+Once the above Pods have successfully started and have begun listening for connections, Kubernetes in each cluster (via automatic health checks) will report them as healthy endpoints of the service in that cluster.  The Cluster Federation will in turn consider each of these service 'shards' to be healthy, and place them in serving by automatically configuring corresponding public DNS records.  You can use your preferred interface to your configured DNS provider to verify this.  For example, if your Federation is configured to use Google Cloud DNS, and a managed DNS domain 'example.com':
```

Comment:
More style: we should decide if shard is a term we're going to use and then decide on whether to scare quote it or not.

We need better terminology than what we've got right now, which is often cumbersomely throwing around modifiers like "underlying", "within the federation", or "in each cluster".